torch/quantization/observer.py

```diff
@@ -533,8 +533,6 @@ def forward(self, x_orig):
             min_val_cur, max_val_cur = torch._aminmax(x)
             min_val = min_val + self.averaging_constant * (min_val_cur - min_val)
             max_val = max_val + self.averaging_constant * (max_val_cur - max_val)
-        self.min_val.resize_(min_val.shape)
```

Comment:
can we keep this and gate behind an if statement, like we did for `FakeQuantize?

Reply:
For this observer we will always have the min/max val have a shape of 1, I believe. We have a separate one - `MovingAveragePerChannelMinMaxObserver` which has different shape.